### ISSUE TITLE
Support Mlflow as LLM-Provider

### ISSUE THREAD

**Issue by essamabas** [Mintplex-Labs/anything-llm#486]
is it planned to support MlFlow as LLM-Provider ?

**Comment by timothycarambat** [comment]
MLFlow is for training models as I understood. Do they provide a normalized API for running inferencing as well?

**Comment by essamabas** [comment]
Yes, MLFlow is used or training/ evaluationg as well as serving Models.
It has unified APIs to return Models with their End-Points https://mlflow.org/docs/latest/llms/deployments/api.html#/

I can work on an extension and provide with pull request.

**Comment by Jayasankar-kk** [comment]
are there plans to add mlflow as an llm provider now
